client/landing/stepper/declarative-flow/copy-site.tsx

```diff
@@ -145,12 +142,6 @@ const copySite: Flow = {
 				}
 
 				case 'processing-copy': {
-					const processingResult = params[ 0 ] as ProcessingResult;
-
-					if ( processingResult === ProcessingResult.FAILURE ) {
-						// @TODO:Create a retry step
-						return navigate( 'retry' );
-					}
```

Comment:
Why is this being removed?

Reply:
This was added initially as a first step to implement a retry step. It's not actually functional. However, we took another approach on resuming implementation, so this code is no longer necessary